xml/System.Text.Json.Serialization/ReferenceHandling.xml

```diff
@@ -55,9 +68,38 @@
         <ReturnType>System.Text.Json.Serialization.ReferenceHandling</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Metadata properties will be honored when deserializing JSON objects and arrays into reference types and written when serializing reference types. This is necessary to create round-trippable JSON from objects that contain cycles or duplicate references.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+* On Serialize:
+ When writing complex reference types, the serializer also writes metadata properties (`$id`, `$values`, and `$ref`) within them.
+ The output JSON will contain an extra `$id` property for every object, and for every enumerable type the JSON array emitted will be nested within a JSON object containing an `$id` and `$values` property.
+ <xref:System.Object.ReferenceEquals(System.Object,System.Object)> is used to determine whether objects are identical.
+ When an object is identical to a previously serialized one, a pointer (`$ref`) to the identifier (`$id`) of such object is written instead.
+ No metadata properties are written for value types.
+ * On Deserialize:
+ The metadata properties within the JSON that are used to preserve duplicated references and cycles will be honored as long as they are well-formed**.
+ For JSON objects that don't contain any metadata properties, the deserialization behavior is identical to <xref:System.Text.Json.Serialization.ReferenceHandling.Default>.
+ For value types:
+ * The `$id` metadata property is ignored.
+ * A <xref:System.Text.Json.JsonException> is thrown if a `$ref` metadata property is found within the JSON object.
+ * For enumerable value types, the `$values` metadata property is ignored.
+ ** For the metadata properties within the JSON to be considered well-formed, they must follow these rules:
```

Comment:
With proper line breaks and indentation, this should look OK as the start of a new paragraph/list without asterisks at the start.

```suggestion
 For the metadata properties within the JSON to be considered well-formed, they must follow these rules:
```